samples/nrf9160/at_client/prj.conf

```diff
@@ -15,3 +15,4 @@ CONFIG_LOG=n
 CONFIG_LOG_DEFAULT_LEVEL=4
 CONFIG_HEAP_MEM_POOL_SIZE=1024
 CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_AT_CMD_INTERFACE=y
```

Comment:
This should be done automatically when selecting `AT_HOST_LIBRARY`.

Reply:
It's is actually selected automatically, so this is probably just residual code. Will remove it.